fxevent/event.go

```diff
@@ -56,14 +50,9 @@ type LifecycleHookStop struct {
 	CallerName string
 }
 
-// ProvideError is emitted whenever there is an error applying options.
-type ProvideError struct {
-	Err error
-}
-
-// Supply is emitted whenever a Provide was called with a constructor provided
+// Supplied is emitted whenever a Provide was called with a constructor provided
 // by fx.Supply.
-type Supply struct {
+type Supplied struct {
```

Comment:
Is there a reason why some of the types here are past tense and some of these are present tense?

Reply:
You might want to refer to GO-677 for discussion on that. Tl;dr: present tense is for events that can be fired both before/after somethign happens, and past tense is for events that are fired after something happens.

Reply:
Ah I see, thanks for the clarification :)

Reply:
Maybe we should add that to the package documentation here; if a future contributor came along and added a new event type here, I don't think they would know that there are naming guidelines.